content/docs/user-guide/scripting/script-canvas/_index.md

```diff
@@ -1,21 +1,25 @@
+Script Canvas is a general purpose, visual scripting environment for **Open 3D Engine (O3DE)** that you can use to create runtime logic and scripted behaviors for any entity. Integrated with O3DE's component entity system, Script Canvas offers an approachable and easy-to-read yet powerful environment to author behaviors using the same framework as Lua and C++. With its visual scripting options, you can use Script Canvas to create [scripts](/docs/user-guide/appendix/glossary#scripts) for game logic without needing to know how to code. Take advantage of O3DE messaging systems to produce small, event-driven scripts that don't require complex logic to maintain states.
 
-Script Canvas is also an extensible system. You can build your own reusable Script Canvas functions and, since Script Canvas is based on AzCore, you can create custom Script Canvas nodes that use the serialization, reflection, modularization, and EBus messaging systems.
+Script Canvas is also an extensible system. You can build your own reusable Script Canvas functions from within the Script Canvas Editor, and use Script Canvas nodes that are created automatically from functionality exposed through the [behavior context](/docs/user-guide/appendix/glossary#behavior-context). The behavior context reflects runtime code, making it accessible to scripts by providing bindings to C++ classes, methods, properties, constants, and enums. The behavior context also provides bindings for O3DE's [AZ::Event](/docs/user-guide/engine/az-event) and [EBus](/docs/user-guide/appendix/glossary#ebus) messaging systems, giving you the ability to use Script Canvas nodes to dispatch and handle messages and events.
 
-To enable Script Canvas for O3DE, you must enable the [Script Canvas](/docs/user-guide/gems/reference/script/script-canvas) Gem and its dependencies.
+To extend Script Canvas even further, you can create your own custom nodes ("nodeables") using the powerful autogen capabilities of the [AzAutoGen](/docs/user-guide/engine/autogen) system.
+
+The ease of use of this visual scripting environment does not come at the cost of performance compared to the more traditional Lua model. To improve performance, Script Canvas graphs are translated by the Asset Processor to optimized Lua script.
+
+To use Script Canvas in your O3DE project, you must enable the [Script Canvas](/docs/user-guide/gems/reference/script/script-canvas) Gem and its dependencies.
 
 | Topics | Description |
 | --- | --- |
 | [Get Started Guide](get-started/) | Learn the basic concepts of Script Canvas and the layout of the Script Canvas Editor. |
```

Comment:
Suggest using "interface" instead of "layout" as I imagine "layout" will be used quite a bit in the context of creating a script canvas graph and this documentation is fleshed out.

```suggestion
 | [Get Started Guide](get-started/) | Learn the basic concepts of Script Canvas and the interface of the Script Canvas Editor. |
```